modules/accounts/email.nix

```diff
@@ -44,6 +44,13 @@ let
         '';
       };
 
+      command = mkOption {
+        type = with types; nullOr (either str path);
+        default = null;
+        example = "./signature.sh";
```

Comment:
When giving a relative path here, what is it relative? The `~/.config/neomutt` directory? Would it be possible to have the type be just `path` to encourage people to put their signature script in the Nix store?